src/operations/hull/hull.test.js

```diff
@@ -0,0 +1,122 @@
+const test = require('ava')
+
+const { geom2 } = require('../../geometry')
+
+const hull = require('./hull')
+
+test('hull (single)', t => {
```

Comment:
good tests !

Reply:
Thanks. I actually identified and fixed some issues with those test cases. :)